packages/core/core/src/ParcelConfig.schema.js

```diff
@@ -0,0 +1,117 @@
+// @flow strict-local
+import type {FilePath} from '@parcel/types';
+import type {SchemaEntity} from '@parcel/utils';
+import {validatePackageName} from './loadParcelConfig';
+
+const validatePluginName = (
+  pluginType: string,
+  key: string,
+  relativePath: FilePath
+) => {
+  return (val: string) => {
+    // Skip plugin spread...
+    if (val === '...') return;
+
+    try {
+      validatePackageName(val, pluginType, key, relativePath);
+    } catch (e) {
+      return e.message;
+    }
+  };
+};
+
+const validateExtends = (relativePath: FilePath) => {
+  return (val: string) => {
+    // Skip plugin spread...
+    if (val.startsWith('.')) return;
```

Comment:
Probably not the right comment?

Reply:
You're right this is copy-pasta from the other validate...